src/app/community-page/community-page.component.html

```diff
@@ -28,10 +28,12 @@
         <ds-themed-comcol-page-browse-by [id]="communityPayload.id" [contentType]="communityPayload.type">
         </ds-themed-comcol-page-browse-by>
 
-        <ds-themed-community-page-sub-community-list [community]="communityPayload"></ds-themed-community-page-sub-community-list>
-        <ds-themed-community-page-sub-collection-list [community]="communityPayload"></ds-themed-community-page-sub-collection-list>
+        <ds-themed-community-page-sub-community-list
```

Comment:
Same here, could you undo the changes to this file too?

Reply:
@GauravD2t : The changes to the `community-page.component.html` are still not undone. They still are visible in the files changed tab in GitHub: https://github.com/DSpace/dspace-angular/pull/2275/files